pkg/cluster/config.go

```diff
@@ -34,6 +34,8 @@ const (
 )
 
 var (
+	// SupportedProxyVars is a list of the supported proxy variables
+	SupportedProxyVars = []string{"HTTP_PROXY", "HTTPS_PROXY", "NO_PROXY"}
 	// ClusterWideProxyVars is a map of the global egress proxy variables and values from WMCO's environment
 	ClusterWideProxyVars map[string]string
```

Comment:
Neither of these need to be public variables, see comment lower in the file.

Reply:
SupportedProxyVars is exported for usage in the e2e tests. let me know if this a bad practice